pennylane/qchem/tapering.py

```diff
@@ -293,44 +293,39 @@ def _taper_pauli_sentence(ps_h, generators, paulixops, paulix_sector):
     for ps in _split_pauli_sentence(ps_h, max_size=PAULI_SENTENCE_MEMORY_SPLITTING_SIZE):
         ts_ps += ps_u @ ps @ ps_u  # helps restrict the peak memory usage for u @ h @ u
 
-    wireset = ps_u.wires + ps_h.wires
+    wireset = ps_h.wires + ps_u.wires
```

Comment:
Why is this needed?

Reply:
Using the original Hamiltonian's wires first is better.

Reply:
Can you provide an example of how this works?